src/app/shared/components/sidemenu/sidemenu.component.ts

```
@@ -202,13 +202,13 @@ export class SidemenuComponent implements OnInit {
     const { allowedReportsActions, allowedAdvancesActions } = this.allowedActions;
     const teamOptions = [
       {
-        title: 'Team expense reports',
+        title: 'Expense reports',
         isVisible:
           allowedReportsActions && allowedReportsActions.approve && showTeamReportsPage && !isOnboardingPending,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)